apps/models/base/subdomain.py

```diff
@@ -5,7 +5,8 @@
 class Subdomain(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     subdomain = models.CharField(max_length=53, unique=True)
-    project = models.ForeignKey(settings.PROJECTS_MODEL, on_delete=models.CASCADE, null=True)
+    project = models.ForeignKey(settings.PROJECTS_MODEL, on_delete=models.CASCADE, null=True, blank=True)
+    user_created = models.BooleanField(default=False)
```

Comment:
The more I read the code the less I was willing to propose it, but at first this was confusing.
That's why I'm going to still suggest this:
```suggestion
    is_created_by_user = models.BooleanField(default=False)
```

Reply:
I agree, this is how it should be.